src/classification/neurologicalLevelOfInjury/neurologicalLevelOfInjury.ts

```diff
@@ -23,42 +23,42 @@ export const checkLevelWithoutMotor = (level: SensoryLevel, leftSensoryResult: C
 }
 
 export const checkLevelWithMotor = (exam: Exam, level: SensoryLevel, sensoryResult: CheckLevelResult, variable: boolean): CheckLevelResult => {
-  if (!sensoryResult.continue) {
-    return sensoryResult;
-  } else {
-    const i = SensoryLevels.indexOf(level);
-    const index = i - (levelIsBetween(i,'C4','T1') ? 4 : 16);
-    const motorLevel = MotorLevels[index];
-    const nextMotorLevel = MotorLevels[index + 1];
+  const i = SensoryLevels.indexOf(level);
+  const index = i - (levelIsBetween(i,'C4','T1') ? 4 : 16);
+  const motorLevel = MotorLevels[index];
+  const nextMotorLevel = MotorLevels[index + 1];
 
-    const leftMotorResult = level === 'C4' || level === 'L1' ?
-      checkMotorLevelBeforeStartOfKeyMuscles(exam.left, level, nextMotorLevel, variable) :
-      level === 'T1' || level === 'S1' ?
-        checkMotorLevel(exam.left, motorLevel, motorLevel, variable) :
-        checkMotorLevel(exam.left, motorLevel, nextMotorLevel, variable);
-    const rightMotorResult = level === 'C4' || level === 'L1' ?
-      checkMotorLevelBeforeStartOfKeyMuscles(exam.right, level, nextMotorLevel, variable) :
-      level === 'T1' || level === 'S1' ?
-        checkMotorLevel(exam.right, motorLevel, motorLevel, variable) : // TODO: hot fix
-        checkMotorLevel(exam.right, motorLevel, nextMotorLevel, variable);
+  const leftMotorResult = level === 'C4' || level === 'L1' ?
+    checkMotorLevelBeforeStartOfKeyMuscles(exam.left, level, nextMotorLevel, variable) :
+    level === 'T1' || level === 'S1' ?
+      checkMotorLevel(exam.left, motorLevel, motorLevel, variable) :
+      checkMotorLevel(exam.left, motorLevel, nextMotorLevel, variable);
+  const rightMotorResult = level === 'C4' || level === 'L1' ?
+    checkMotorLevelBeforeStartOfKeyMuscles(exam.right, level, nextMotorLevel, variable) :
+    level === 'T1' || level === 'S1' ?
+      checkMotorLevel(exam.right, motorLevel, motorLevel, variable) : // TODO: hot fix
+      checkMotorLevel(exam.right, motorLevel, nextMotorLevel, variable);
 
-    let resultLevel;
-    if (leftMotorResult.level || rightMotorResult.level || sensoryResult.level) {
-      if (
-        leftMotorResult.level && rightMotorResult.level &&
-        leftMotorResult.level.includes('*') && rightMotorResult.level.includes('*')
-      ) {
-        resultLevel = level + '*';
-      } else {
-        resultLevel = level + (variable ? '*' : '');
-      }
+  let resultLevel;
+
+  if (leftMotorResult.level || rightMotorResult.level || sensoryResult.level) {
+    if (
+      leftMotorResult.level && rightMotorResult.level &&
+      (leftMotorResult.level.includes('*') || rightMotorResult.level.includes('*'))
+    ) {
+      resultLevel = level + '*';
+    } else {
+      resultLevel = level + (variable ? '*' : '');
     }
-    return {
+  }
+
+  return  !sensoryResult.continue

```

Comment:
I moved this check from the top to here so we can make sure we are adding the `*` to the result when needed.